### ISSUE TITLE
Output satellites count comes from wrong sentence

### ISSUE THREAD

**Issue by duncanbarnes** [vasimv/rtkzedf9p#1]
TinyGPS++ takes the number of SV satellites from the GGA sentence, for an RTK solution this should (I think) come from GNS sentence field 7 for the RTKLIB output

I've so far failed to get TinyGPS++ custom NMEA sentence extraction to pick this out although my C++ knowledge isn't great, when I do I'll submit a pull but thought I'd flag it in the interim.

**Comment by vasimv** [comment]
Perhaps you right as i see only "12" number always in RTK mode (although it may be internal filtering for best satellites?).

**Comment by duncanbarnes** [comment]
Yes, I see the same, GNS seems to give a more believable number that matches uCenter's numbers but I will read more.
